docs/proposals/external-service-integration.md

```diff
@@ -0,0 +1,51 @@
+#External Service Integrations
+
+## Background
+https://github.com/kubernetes-incubator/service-catalog/issues/1455
+
+## Abstract
+
+This document aims to provide solutions to the following use cases: 
+
+1) A mobile enabled service has been provisioned via the service catalog into namespace A. It is intended for this to be a shared service. Think something like
+Authentication. 
+An MCP is provisioned in namespace B along with another mobile enabled service that is capable of integrating with the mobile service in namespace A. How do we support
+this kind of integration?
+
+
+2) A user or org has an existing service such as Keycloak that exists entirely separately to OpenShift, but is made accessible. It is the intention that this would be a
+shared service. A developer then creates a mobile project with MCP provisioned on an OpenShift instance, in what circumstances can we provide integration and what form
+should that integration take?
+
+
+## Cross Namespace integrations
+
+We are leveraging the machinery provided by the service catalog to power our integrations. This document assumes some level of familiarity with ServiceBindings and ServiceInstances and the ansible service broker.
+
+Currently a binding can only be created in the same namespace as an existing provisioned a service instance:
+https://github.com/kubernetes-incubator/service-catalog/issues/1455
+
+The end result of a successful binding is a secret, that contains a set of credentials that is then created within the same namespace as the service instance.
```

Comment:
This paragraph is pretty hard to grasp. Maybe it would be easier to read if the goals were broken into bullet points, or a separate paragraph for each? Or explain it using an example?